bin/minifollowups/pycbc_injection_minifollowup

```diff
@@ -140,6 +144,49 @@ except KeyError:
     dec_chirp_dist = pycbc.pnutils.chirp_distance(dec_dist, mchirp)
     sorting = dec_chirp_dist.argsort()  # ascending order of dec chirp distance
 
+# Get the trigger SNRs and times
+# But only ones which are within a small window of the missed injection
+missed_inj_times = numpy.sort(f['injections/tc'][:][missed])
+
+# Note: Adding Earth diameter in light seconds to the window here
+# to allow for different IFO's arrival times of the injection
+safe_window = args.nearby_triggers_window + 0.0425
+
+def nearby_missedinj(endtime, snr):
+    """
+    Convenience function to check if trigger times are within a small
+    window of the injections
+
+    Parameters
+    ----------
+    endtime: numpy array
+    Trigger times to be checked against the missed injection times
+
+    snr: numpy array
+    (Unused, but to prevent a TypeError) SNR of the triggers
```

Comment:
This is cryptic (very unclear why a TypeError might arise). I am guessing that the way the `.select` method is set up requires this. If so, rephrase eg 'Required by design of the HFile.select method but not used'